lib/src/linking.rs

```diff
@@ -102,7 +105,7 @@ pub(crate) fn create_store(
     store.set_epoch_deadline(1);
     store.epoch_deadline_callback(|mut store| {
         if let Some(mut prof) = store.data_mut().guest_profiler.take() {
-            prof.sample(&store);
+            prof.sample(&store, std::time::Duration::ZERO);
```

Comment:
I'm just going to highlight this, because a sample duration of zero kind of confuses me. I guess it can't be the epoch time because we may have taken other samples during that epoch? Is there a reason ("because it's too slow, Adam") to not keep a time-since-last-sample, or something?

Reply:
This delta parameter is the amount of cpu time that was taken since the last `sample` call. Passing zero is called out as okay in the docs for cases where cpu usage information isn't needed. We can always start tracking that for more precise profiles in the future, but this seemed like a reasonable first step :)